src/redux/categories/categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div className="w-100">
+    <h1 className="text-align-self">Site Under Construction</h1>
+  </div>
+);
+
+export default Categories;
```

Comment:
- [ ] Move this component to the components directory.

Reply:
But according to this, `categories.js` should be inside the Categories folder. In that case, should also move the `Books` components? Oh, or those folders are only for Redux files? 

![](https://i.ibb.co/g34PvHq/Captura.jpg)

Reply:
Those files in the redux folder will contain the redux logic only i.e actions and action types.